src/scss/styles.scss

```diff
@@ -123,6 +123,18 @@ body {
     &.layout_frontend {
         background-color: #ecf0f5;
     }
+
+    &.full-width {
+        .container {
+            min-width: 980px;
+            width: 90%;
+            .row {
+                .col-5 {
+                    max-width: 416px;
+                }
```

Comment:
col-5 max-width set to keep the login box the same size regardless of weather on not the fill width setting is turned on.

Reply:
Let's remove the override for the login form itself, if we need to reset it I would prefer it get it's own "functional class" for css targeting (e.g. "`.login-form`") but looking at it, while it's certainly a lot wider in this scenario, I believe it's acceptable without limiting its size.